cloudsql-postgresql-plugin/pom.xml

```diff
@@ -102,6 +102,12 @@
       <version>42.3.1</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>com.google.code.gson</groupId>
```

Comment:
Why do we need this ?

Reply:
This is basically used to convert java objects into json representation and vice versa so that's why we are using this.

Reply:
Its already present in google-http-client dependency which is added in cdap-e2e-test repo. Use ```import com.google.gson.Gson``` and try if you are able to resolve it without adding a extra dependency for Gson.